16 error Error: Command failed with exit code 1: git remote update
16 error fatal: unable to access 'https://github.com/zhangheng0317/zhall.git/': Failed to connect to github.com port 443 after 21070 ms: Timed out
16 error error: Could not fetch origin
16 error Fetching origin
16 error     at makeError (C:\ENV\Node\node_global\node_modules\lerna\node_modules\execa\lib\error.js:60:11)
16 error     at Function.module.exports.sync (C:\ENV\Node\node_global\node_modules\lerna\node_modules\execa\index.js:194:17)
16 error     at Object.execSync (C:\ENV\Node\node_global\node_modules\lerna\node_modules\@lerna\child-process\index.js:39:16)
16 error     at updateRemote (C:\ENV\Node\node_global\node_modules\lerna\node_modules\@lerna\version\lib\is-behind-upstream.js:34:16)
16 error     at isBehindUpstream (C:\ENV\Node\node_global\node_modules\lerna\node_modules\@lerna\version\lib\is-behind-upstream.js:16:3)
16 error     at VersionCommand.initialize (C:\ENV\Node\node_global\node_modules\lerna\node_modules\@lerna\version\index.js:153:9)
